docs/concepts/automations.md

```diff
@@ -310,6 +314,86 @@ However, without `for_each`, a `prefect.flow-run.Completed` event from a _differ
 }
 ```
 
+
+### Metric triggers
+
+Metric triggers (`{"type": "metric"}`) fire when the value of a metric in your workspace crosses a threshold you've defined.  For example, you can trigger an automation when the success rate of flows in your workspace drops below 95% over the course of an hour.
+
+Prefect's metrics are all derived by examining your workspace's event stream, and if applicable, use the `occurred` times of those events as the basis for their calculations.
+
+Prefect defines three metrics today:
+
+* **Successes** (`{"name": "successes"}`), defined as the number of flow runs that went `Pending` and then the latest state we saw was not a failure (`Failed` or `Crashed`).  This metric accounts for retries if the ultimate state was successful.
+* **Duration** (`{"name": "duration"}`), defined as the _length of time_ that a flow remains in a `Running` state before transitioning to a terminal state like `Completed`, `Failed`, or `Crashed`.  Because this time is derived in terms of flow run state change events, it may be greater than the runtime of your function.
+* **Lateness** (`{"name": "lateness"}`), defined as the _length of time_ that a `Scheduled` flow remains in a `Late` state before transitioning to a `Running` and/or `Crashed` state.  Only flow runs that the system marks `Late` are included.
+
+The schema of a metric trigger is as follows:
+
+| Name               | Type               | Supports trailing wildcards | Description |
```

Comment:
Looks like my table got a little scrambled on me, do you all have any VSCode extension/config recommendations for Markdown tables?

Reply:
I went with the "Markdown All in One" package and it seems great